classList.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Element.classList utility functions
+ * @module delite/classlist
+ * */
+define([], function () {
+
+	var classList = /** @lends module:delite/classList */ {
+		/**
+		 * Toggle one or multiple classes helper method.
+		 *
+		 * @param {Element} node
+		 * @param {String} value
+		 * @param {Boolean} force
+		 */
+		toggleClass: function (node, value, force) {
+			if (value && value.trim() !== "") {
+				if (force === true) {
+					classList.addClass(node, value);
+				}
+				else if (force === false) {
+					classList.removeClass(node, value);
+				}
+				else {
+					var method;
+					this._getValues(value).forEach(function (v) {
+						method = node.classList.contains(v) ? "removeClass" : "addClass";
+						classList[method](node, v);
+					}, this);
+				}
+			}
+		},
+
+		/**
+		 * Add one or multiple classes.
+		 *
+		 * @param {Element} node
+		 * @param {String} value Single or space-separated string representing the classes to be added.
+		 */
+		addClass: function (node, value) {
+			if (value && value.trim() !== "") {
+				node.classList.add.apply(node.classList, this._getValues(value));
```

Comment:
Multiple args don't work on IE.  Need a loop.  Likewise for removeClass().

Reply:
Fixed.